Stubs/.cfnlintrc

```diff
@@ -1,8 +1,10 @@
 templates:
 - ./**/*.yml
-#- ./**/*.yaml
+- ./**/*.yaml
 ignore_templates:
 - ./**/buildspec.yml
-- ./**/buildspec.yaml
-- ./**/buildspec*.yml
-- ./**/buildspec*.yaml
+#- ./**/buildspec.yaml
+#- ./**/buildspec*.yml
+#- ./**/buildspec*.yaml
+ignore_checks:
+- W3045 # Ignoring this check because 'AccessControl' should still be supported for our use case
```

Comment:
Need to remove templates mentioned in ignored that are not existing in repository. Also needed the same file in root directory